### DISCUSSION TITLE
TTS possibility

### DISCUSSION THREAD

**Discussion by timvdsm** [punxaphil/custom-sonos-card#154] [Ideas]
Hi, first of all, this card is absolutely amazing! 
Loving it every day 👍🏻 and great to see all those nice features implemented 👏🏻.

I just wanted to check how you guys use Text-To-Speech in combination with sonos speakers. I want to send TTS messages to my sonos speakers through the house. 
I see TTS is now implemented in the official Media Library in the media entity itself. However then you need to go to the official media entity, .... which is not very user friendly.
Is there any way to implement this in an easy and friendly way in the sonos card? And to select a speaker/group of speakers to send the TTS to? How do you guys use this? 

Thx!

**Comment by punxaphil** [comment]
Nice idea, I won't be able to do it though anytime soon. I'll tag this with "help wanted"
